srcs/utils_tx.c

```diff
@@ -0,0 +1,26 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   utils_tx.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gudias <marvin@42lausanne.ch>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/09/22 12:27:25 by gudias            #+#    #+#             */
+/*   Updated: 2022/09/22 12:32:50 by gudias           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "cub3d.h"
+
+t_img_data	*get_wall_tx(t_info *info, char wall)
```

Comment:
> **For a future update**

I got an idea to remove this function:
1. In `collect data ray` use enum instead character:

   ```c
   info->ray[column].wall = NO;
   info->ray[column].wall = SO;
   ...
   ```
3. Use ray.wall in `[ ]`:

   ```c
   info->texture[info->ray[column].wall].img
   ```